admin_pages/maintenance/templates/ee_migration_was_borked_page.template.php

```diff
@@ -71,16 +71,16 @@
             </td>
             <td>
                 <textarea name="body" id="body" class='system_status_info'>
-                    <?php esc_html_e(
+                    <?php esc_textarea(__(
                         "Enter any comments about why you think the error may have occurred",
                         "event_espresso"
-                    ); ?>
+                    )); ?>
```

Comment:
Good choice 👍🏻 forgot about that function. 

lolz... looked that up in the wp docs to see what it did and the only user comment is from a previous EE dev:
https://developer.wordpress.org/reference/functions/esc_textarea/

Reply:
👍